resources-generator/src/main/kotlin/dev/icerock/gradle/generator/apple/AppleMRGenerator.kt

```diff
@@ -96,21 +96,14 @@ class AppleMRGenerator(
         setupFrameworkResources()
         setupTestsResources()
         setupFatFrameworkTasks()
-
-        dependsOnProcessResources(
-            project = project,
-            sourceSet = sourceSet,
-            task = generationTask,
-            shouldExcludeGenerated = true
-        )
     }
 
     override fun beforeMRGeneration() {
         assetsDirectory.mkdirs()
     }
 
     private fun setupKLibResources(generationTask: Task) {
-        val compileTask: KotlinNativeCompile = compilation.compileKotlinTask
+        val compileTask: KotlinNativeCompile = compilation.compileTaskProvider.get()
```

Comment:
we should not get task at configuration time. please set dependencies with taskProvider, without call `get`